### PR TITLE
Switched music paths to ensure windows compatibility

### PR DESCRIPTION
**Pull request by lxgr-linux**
Please test @MaFeLP .

### CODE REVIEW

**Review by MaFeLP — CHANGES_REQUESTED** [review]
Does not work on every map.

Tested works in `Pokete Center`.\
Does not work in `NiceTown` or `Fightmap`:

***

Stacktrace for NiceTown
```log
    Error 305 for command:
        open "C:\Users\maxfe\AppData\Local\Temp\PS_4klm1ho.mp3"
    Cannot specify extra characters after a string enclosed in quotation marks.

    Error 263 for command:
        close "C:\Users\maxfe\AppData\Local\Temp\PS_4klm1ho.mp3"
    The specified device is not open or is not recognized by MCI.
Failed to close the file: "C:\Users\maxfe\AppData\Local\Temp\PS_4klm1ho.mp3"
Process Process-3:
Traceback (most recent call last):
  File "C:\Users\maxfe\AppData\Local\Programs\Python\Python310\lib\multiprocessing\process.py", line 315, in _bootstrap
    self.run()
  File "C:\Users\maxfe\AppData\Local\Programs\Python\Python310\lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\maxfe\pokete\pokete_classes\audio.py", line 18, in audio_fn
    playsound(MUSIC_PATH / song)
  File "C:\Users\maxfe\AppData\Local\Programs\Python\Python310\lib\site-packages\playsound.py", line 44, in _playsoundWin
    _playsoundWin(tempPath, block)
  File "C:\Users\maxfe\AppData\Local\Programs\Python\Python310\lib\site-packages\playsound.py", line 72, in _playsoundWin
    winCommand(u'open {}'.format(sound))
  File "C:\Users\maxfe\AppData\Local\Programs\Python\Python310\lib\site-packages\playsound.py", line 64, in winCommand
    raise PlaysoundException(exceptionMessage)
playsound.PlaysoundException:
    Error 305 for command:
        open "C:\Users\maxfe\AppData\Local\Temp\PS_4klm1ho.mp3"
    Cannot specify extra characters after a string enclosed in quotation marks.
```

**Comment by lxgr-linux** [comment]
WTF

File: pokete_classes/audio.py [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
+from pathlib import Path
 try:
     from playsound import playsound
 except ModuleNotFoundError:
     from .dummy_playsound import playsound
 from .settings import settings
 
 
-MUSIC_PATH = __file__.replace("pokete_classes/audio.py", 'assets/music/')
+MUSIC_PATH = Path(__file__).parent.parent / 'assets' / 'music'
```
Comment (mustafaquraish) [inline pokete_classes/audio.py:12]: just FYI, should be able to do `Path(__file__).parents[1]`

Comment (lxgr-linux) [reply pokete_classes/audio.py]: Oh yes, that's a good tip.

**Comment by lxgr-linux** [comment]
> "C:\Users\maxfe\AppData\Local\Temp\PS_4klm1ho.mp3"

I wonder where this path comes from.

**Comment by lxgr-linux** [comment]
And there seams to be a problem with playsound: https://github.com/TaylorSMarks/playsound/issues/98

**Comment by lxgr-linux** [comment]
Best option would be to enforce `playsound=1.2.2`, the best alternative pydub needs ffmpeg installed, which would be a hassle for the windows users.

**Comment by lxgr-linux** [comment]
I now went through the pain of setting up a windows vm and finally fixing
audio for windows. Also take a look at the new installation instructions @MaFeLP .

**Comment by lxgr-linux** [comment]
It now does not rely on pynput anymore. At all.

**Comment by lxgr-linux** [comment]
This will also fix #217 and #124.

**Comment by MaFeLP** [comment]
Review status:

- [x] Music working on Windows
- [x] keys not on terminal after exit on Windows[^1]
- [ ] Music working on macOS
- [ ] keys not on terminal after exit on macOS

[^1]: Interesting observation: when running `.\pokete.py` instead of `python \.pokete.py`, pokete will be launched in a separate window!

**Comment by lxgr-linux** [comment]
> 1. Interesting observation: when running `.\pokete.py` instead of `python \.pokete.py`, pokete will be launched in a separate window! [leftwards_arrow_with_hook](#user-content-fnref-1-ad8295bfc50b451b025d14d47732a3da)

That looks like common windows BS and may not be related to Pokete at all.

File: pokete.py [hunk 2]
```diff
@@ -1389,9 +1382,30 @@ def map_additions():
...
-    do_logging, load_mods, force_pynput = parse_args(sys.argv)
+    do_logging, load_mods = parse_args(sys.argv)
     # deciding on wich input to use
-    if sys.platform == "linux" and not force_pynput:
+    if sys.platform == "win32":
+        import msvcrt
+
+
+        def recogniser():
+            """Gets keyboard input from pynput"""
```
Comment (MaFeLP) [inline pokete.py:1392]:
```suggestion
            """Gets keyboard input from msvcrt, the Microsoft Visual C++ Runtime"""
```

**Review by MaFeLP — CHANGES_REQUESTED** [review]
Apart from this one comment, everything looks good!

File: pokete.py (hunk 2 above)
Comment (lxgr-linux) [reply pokete.py]: Oh yes, I forgot about the comments.
Then please approve.

File: README.md [hunk 3]
```diff
@@ -22,7 +22,7 @@
 Pokete is a small terminal based game in the style of a very popular and old game by Gamefreak.
 
 ## Installation
-For Linux, run this:
+For Linux, BSD, OSX etc. just do this:
 ```shell
 # pip install scrap_engine playsound pygobject
```
Comment (MaFeLP) [inline README.md:27]: `pygobject` does not work on macOS. Use `pyobjc` on macOS instead.

**Review by MaFeLP — CHANGES_REQUESTED** [review]

**Review by MaFeLP — APPROVED** [review]

**Pull request merged**